src/redux/configureStore.js

```diff
@@ -0,0 +1,11 @@
+import { createStore, combineReducers } from 'redux';
+import reducerForCategory, { createActionForCategory } from './categories/categories';
+import reducer, { createActionForBookToAdd } from './books/books';
+
+const rootReducer = combineReducers({ books: reducer, categories: reducerForCategory });
+
+const store = createStore(rootReducer);
+store.dispatch(createActionForCategory());
+store.dispatch(createActionForBookToAdd());
```

Comment:
- [ ] These 2 dispatchers are not needed, since you will use `dispatch` in a later milestone, from the components.  Kindly remove these 2 dispatchers.